tensornetwork/tests/network_operations_test.py

```diff
@@ -274,3 +274,37 @@ def test_full_graph_subgraph_dangling(backend):
   b[1] ^ c[1]
   edges = tn.get_subgraph_dangling({a, b, c})
   assert edges == {a[1], c[0]}
+
+
```

Comment:
Can you add a test where we do the final contraction and materialise the reduced density matrix and check that it's what you expect?

Reply:
Sure. Could you elaborate or maybe point me to some sample code about final contraction and materialization? Thanks!

Reply:
Don't worry about it. I'll push a test.

Reply:
Thanks. Please cc'd me the PR. I am interested in that too.

Reply:
I plan on pushing it on this PR. ;)